src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,39 @@
+package duke.tasks;
+
+import duke.Parser;
+
+import java.time.LocalDateTime;
+
+public class Deadline extends Task {
+
+    static protected String DEFAULT_TIME = "2359";
+    protected LocalDateTime by;
```

Comment:
Not sure if others will understand what the `by` variable represents on first look! Perhaps you could consider renaming it to `dateBy` so as to convey to others that it is a date!